src/middlewared/middlewared/plugins/smb_/status.py

```diff
@@ -61,7 +62,21 @@ async def status(self, info_level, filters, options, status_options):
                 return ret
 
             for e in logfile_entries.splitlines():
-                ret.append(json.loads(e.strip()))
+                # sample timestamp entry:
+                # "2021-12-31T05:42:33.954029-0800"
+                entry = json.loads(e.strip())
+                ts, extra = entry['timestamp'].split('.', 1)
+                # add timezone info
+                ts += extra[6:]
+                usec = extra[:6]
```

Comment:
```
ts = talloc_asprintf(frame, "%s.%06ld %s", buffer, tv.tv_usec, tz);
```

Reply:
It appears to not be the case:
```
    "timestamp": "2022-02-17T00:41:54.998990-0800",
    "type": "Authentication",
```
But my brain has bit-rotted since I wrote this part of the alert, and so I'll review Samba code for how it's generating this timestamp and how I'm converting to timeval.